src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -189,6 +189,43 @@ class ContentSidebar extends PureComponent<Props, State> {
         onInteraction(Object.assign({}, { file: cloneDeep(file) }, data));
     };
 
+    /**
+     * Function to update file description
+     *
+     * @private
+     * @param {string} value - new file description
+     * @return {void}
+     */
+    onDescriptionChange = (value: string): void => {
+        const { file } = this.state;
+        if (!file) {
+            return;
+        }
+
+        const { description, id } = file;
+        if (value === description || !description || !id) {
```

Comment:
If original description was empty, it will not update it to a new one.

Reply:
It looks like it gets called no matter what onblur

```   onBlur={ event => {
            onDescriptionChange(event.target.value);
        } }
```